resources/assetstore/charts/minio/templates/secrets.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     heritage: {{ .Release.Service }}
 type: Opaque
 data:
-  accesskey: {{ .Values.accessKey | b64enc }}
-  secretkey: {{ .Values.secretKey | b64enc }}
+  accesskey: {{ randAlphaNum 40 | b64enc | quote }} 
```

Comment:
It should be possible to provide a custom value. So please make it like here: 
https://github.com/helm/charts/blob/master/stable/grafana/templates/secret.yaml#L14
or like here:
https://github.com/kyma-project/kyma/blob/623af05f4401f4c4d3d691f5df9939bb577a023e/resources/assetstore/charts/minio/templates/content-upload-configmap.yaml#L16